src/storage/store/column_chunk_metadata.cpp

```diff
@@ -128,7 +143,7 @@ ColumnChunkMetadata getConstantFloatMetadata(PhysicalTypeID physicalType, uint64
 template<std::floating_point T>
 alp::state getAlpMetadata(const T* buffer, uint64_t numValues) {
     alp::state alpMetadata;
-    std::vector<uint8_t> sampleBuffer(alp::config::SAMPLES_PER_ROWGROUP);
+    std::vector<uint8_t> sampleBuffer(alp::config::SAMPLES_PER_ROWGROUP * sizeof(T));
```

Comment:
Since this is just being used by casting it to `T*` on the next line, why not have it be a `std::vector<T>` instead?

Reply:
updated